script/repository-list.txt

```diff
@@ -14,3 +14,4 @@ jbpm-designer
 jbpm-wb
 kie-docs
 kie-wb-distributions
+jbpm-work-items
```

Comment:
I think this should go directly after the `jbpm` repo.

Reply:
I put it as last one in as nothing else depends on it as the intention is whatever goes here should not be bundled with any app (other repository module). It should be purely available though service repository. So that way we could catch it before we release - as build should fail if something reference any work item from this repo. Does it make sense?

Reply:
Makes sense.